jax/_src/nn/functions.py

```diff
@@ -165,7 +165,7 @@ def log_sigmoid(x: Array) -> Array:
   return -softplus(-x)
 
 @jax.jit
-def elu(x: Array, alpha: Array = 1.0) -> Array:
```

Comment:
Array inputs in this module should be annotated `ArrayLike`, otherwise we're likely to have spurious errors if someone calls these functions on e.g. a numpy array or scalar

Reply:
Done, PTAL.